src/modules/LanUdevAuto.rb

```diff
@@ -205,7 +207,7 @@ def Write
       end
 
       if !rules.empty? && AllowUdevModify()
-        SetAllLinksDown()
+        SetAllLinksDown() if !Linuxrc.usessh
```

Comment:
it is. But this is a quick fix which breaks another use case so I've limited that only on reported case.  Another improvement in this patch is a must.